libs/coin-modules/coin-aptos/src/__tests__/api/index.integ.test.ts

```diff
@@ -21,4 +23,35 @@ describe("createApi", () => {
   //     expect(hash).toEqual(expect.any(String));
   //   });
   // });
+
+  describe("estimateFees", () => {
+    it("returns a default value", async () => {
+      // Account has been funded using the await aptos.fundAccount method
+      const sender: AptosSender = {
+        xpub: "0xc7a5a529eb69b4f40519c3334eed48f090af354ec6c0893129ebed30328e245c",
+        freshAddress: "0xf2a89ea976c60f98bd8a2cbc33b49e6d1de38618a6991692dee61c9b4745ad4a",
+      };
+      const recipient = Account.generate().accountAddress.toString();
+
+      const amount = BigInt(100);
+
+      const fees = await api.estimateFees({
+        asset: {
+          type: "native",
+          function: "0x1::aptos_account::transfer_coins",
+        },
+        type: "send",
+        sender,
+        amount,
+        recipient,
+      });
+
+      expect(fees.value).toEqual(BigInt(55100));
+
+      if (fees.parameters) {
```

Comment:
To be removed, no conditions in test

We setup context and expect outputs everytime